.github/workflows/ci.yml

```diff
@@ -7,21 +7,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
-        numpy: [null, "numpy>=1.17,<2.0.0"]
-        uncertainties: [null, "uncertainties==3.1.4", "uncertainties>=3.1.4,<4.0.0"]
+        python-version: [3.8, 3.9, "3.10"]
+        numpy: [null, "numpy>=1.19.5,<2.0.0"]
```

Comment:
is there a reason for requiring `numpy>=1.19.5` instead of `numpy>=1.19`? Or is this is from the min-versions job?

Reply:
No just want to be sure to use a numpy patched version.